web/__init__.py

```diff
@@ -1,61 +1,83 @@
 import json
+import tempfile
 import logging as log
 
-from flask import Flask, request, render_template, Response
+from secrets import token_urlsafe as token
+from flask import Flask, request, render_template, Response, session, url_for
 from flask_sock import Sock
-
-from libflagship.pppp import P2PSubCmdType, FileTransfer
-from libflagship.ppppapi import FileUploadInfo, PPPPError
-
+from user_agents import parse as user_agent_parse
 
 from web.lib.service import ServiceManager
 
+import web.config
+import web.platform
+import web.util
+
 import cli.util
+import cli.config
+
 
 app = Flask(
     __name__,
     root_path=".",
     static_folder="static",
     template_folder="static"
 )
+# secret_key is required for flash() to function
+app.secret_key = token(24)
 app.config.from_prefixed_env()
 app.svc = ServiceManager()
 
 sock = Sock(app)
 
-
-import web.service.pppp
-import web.service.video
-import web.service.mqtt
-import web.service.filetransfer
+import web.service.pppp  # nopep8
```

Comment:
What reacts to this syntax? If it's only vscode, please remove it.

Reply:
it prevents vscode and any other IDE that uses pep8 for python formatting, to not move around imports all willy nilly. That is what broke the app for me that led me to moving the imports in the services. I reverted that so had to add this here to keep my IDE from fuxing me again :D

Reply:
✅